docs/2.getting-started/1.installation.mdx

```diff
@@ -55,7 +55,7 @@ In order to develop with Botonic on Windows, you can achieve it by following thi
 
 **Requirements:**
 
-* **[NodeJS](https://nodejs.org/en/download/)**: tested on v8.12.0
+* **[NodeJS](https://nodejs.org/en/download/)**: tested on v11.9.0
```

Comment:
It doesn't work with previous versions?

Reply:
It works, maybe it would be better to say "tested up to v11.9.0"

Reply:
Explicit is better than implicit. Let's put `tested on v8.12, [...] and 11.9`